SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialNetDriver.cpp

```diff
@@ -493,21 +476,22 @@ void USpatialNetDriver::CreateAndInitializeLoadBalancingClasses()
 	LoadBalanceStrategy->SetVirtualWorkerIds(1, LoadBalanceStrategy->GetMinimumRequiredWorkers());
 
 	VirtualWorkerTranslator = MakeUnique<SpatialVirtualWorkerTranslator>(LoadBalanceStrategy, Connection->GetWorkerId());
+	VirtualWorkerTranslator->SetNetDriver(this);
```

Comment:
Why not just pass it in the constructor?

Reply:
Will do